device/rsvp/rsvpEgressLsp.yaml

```diff
@@ -0,0 +1,49 @@
+components:
+  schemas:
+    Rsvp.LspInterface.EgressP2PLSP:
+      description: >-
+        Configuration for RSVP Egress P2P LSPs.
+      type: object
+      properties:
+        refresh_interval:
+          description: >-
+            The time in seconds between transmissions of RESV Refreshes.
+            The actual refresh interval is jittered by upto 50%.
+            There is no specification specified maximum value. For clarity, setting the maximum to 1 hour. 
+          type: integer
+          default: 30
+          minimum: 1
+          maximum: 3600
+        timeout_multiplier:
+          description: >-
+            The number of missed RESV refreshes after which a recieving node should consider the LSP state to have timed out.
+            There is no specification specified maximum value. Setting the maximum allowed value to 10 .
+          type: integer
+          default: 3
+          minimum: 1
+          maximum: 10
+        reservation_style:
+          description: >-
+            It determines how RSVP-TE enabled network devices set up reservations along the path between an end-to-end 
+            QOS-enabled connection. If 'auto' is enabled, the style is chosen based on the whether incoming Path has 'SE Desired'
+            flag set. Otherwise, the style is chosen based on the value selected for this attribute.
+          type: string
+          enum:
+          - shared_explicit
+          - fixed_filter
+          - auto
+          default: shared_explicit
+        enable_fixed_label:
+          description: >-
+            If enabled, a specific label will be advertised by the egress or tail end.This can be leveraged to advertise Explicit or Implicit null labels.            
+          type: boolean
+          default: false
+        fixed_label_value:
+          description: >-
+             The fixed label value as advertised by egress in RESV message.Applicable only if 'fixed_label' is set to 'true'.
+             Special values are '0 - IPv4 Explicit NULL' ,'2 - IPv6 Explicit NULL' and '3 - Implicit NULL',.  
```

Comment:
remove the comms before full stop.

Reply:
Looks like stale file version. New files were uploaded yesterday but the review seems to be showing the old versions.